config/locales/models/activity.en.yml

```diff
@@ -75,6 +83,21 @@ en:
         parent: Select the parent activity
         planned_end_date: Planned end date
         planned_start_date: Planned start date
+        policy_markers:
```

Comment:
I might be misunderstanding something, but could we not use the codelist from https://iatistandard.org/en/iati-standard/203/codelists/policymarker/ for this?

Reply:
I _think_ we don't use the IATI codelists on the locales files, only in the vendor. `activity.en.yml` is for translation files for the UI. It's a RODA thing rather than an IATI thing, if you know what I mean?